src/test/java/io/github/enkarin/bookcrossing/books/service/BookServiceTest.java

```diff
@@ -43,39 +46,61 @@ void saveExceptionTest() {
                 .hasMessage("Пользователь не найден");
     }
 
-    @SqlGroup({
-        @Sql("classpath:db/scripts/insert_user.sql"),
-        @Sql("classpath:db/scripts/insert_books.sql")
-    })
     @Test
     void findBookForOwnerTest() {
+        final User user1 = userService.saveUser(UserDto.create("Tester", "user", "123456",
```

Comment:
Consider to use https://projectlombok.org/features/experimental/SuperBuilder on DTO's and in tests

Reply:
Added